google-cloud-tools-plugin/ultimate/src/com/google/cloud/tools/intellij/appengine/server/run/CloudSdkStartupPolicy.java

```diff
@@ -67,13 +66,11 @@ public String getDisplayString() {
           @Override
           public OSProcessHandler createProcessHandler(
               String workingDirectory, Map<String, String> envVariables) throws ExecutionException {
-            CloudSdkService sdkService = CloudSdkService.getInstance();
 
-            if (sdkService.getSdkHomePath() == null
-                || sdkService.getSdkHomePath().toString().isEmpty()) {
+            CloudSdkService sdkService = CloudSdkService.getInstance();
+            if (sdkService.getSdkHomePath() == null) {
               throw new ExecutionException(
-                  CloudSdkPanel.createErrorMessageWithLink(
-                      GctBundle.message("appengine.cloudsdk.location.missing.message")));
+                  GctBundle.message("appengine.run.server.sdk.misconfigured.message"));
```

Comment:
what would I attach the browser listener to? I typically do it by attaching to the UI component, which I don't seem to have access to here